src/components/elements/checkbox/README.md

```diff
@@ -20,4 +20,14 @@ input[type="checkbox"] + label::before {
 ```
 
 Labels and the corresponding checkbox element are intended to be wrapped in a
-containing element of some sort, ideally a paragraph.
+containing element with a class of `field`.
+
+A hint can be added immediately after the `<label>` in a paragraph with
+a class of `field__hint`.
+
+A validation error message can be added immediately after the `<label>`
+or hint in a paragraph with a class of `field__error`.
+
+Hints and validation error messages must be associated with the checkbox using
+unique IDs which can be referenced from the checkbox's `aria-describedby`
+attribute. For more information, see [here](https://webaim.org/techniques/formvalidation/#error).
```

Comment:
In all the docs, rather than have a link labeled "here" it might be more helpful to have a link that identifies the resource. like "WebAim form validation error recovery" or something like that.

Reply:
Fixed in 0168518.